qemu/tests/cfg/virtio_fs_supp_group_transfer.cfg

```diff
@@ -0,0 +1,51 @@
+- virtio_fs_supp_group_transfer:
+    type = virtio_fs_supp_group_transfer
+    only Linux
+    no Host_RHEL.m8, Host_RHEL.m7, Host_RHEL.m6
+    virt_test_type = qemu
+    required_qemu = [4.2.0,)
+    s390, s390x:
+        required_qemu = [5.2.0,)
+        vm_mem_share = yes
+        pre_command_noncritical = yes
+        pre_command = "echo 3 > /proc/sys/vm/drop_caches"
+        setup_hugepages = yes
+        kvm_module_parameters = 'hpage=1'
+        expected_hugepage_size = 1024
+    clone_master = yes
+    master_images_clone = image1
+    remove_image_image1 = yes
```

Comment:
if we don't need these, we can remove them, refer to log:
[stdlog]     -blockdev '{"node-name": "file_image1", "driver": "file", "auto-read-only": true, "discard": "unmap", "aio": "threads", "filename": "/home/kvm_autotest_root/images/rhel930-64-virtio-scsi-ovmf.qcow2", "cache": {"direct": true, "no-flush": false}}' \

So, the cloned image was not used at all